app/views/shared/_search_form.html.erb

```diff
@@ -1,3 +1,5 @@
+<%# locals: (url:, search_param:, clear_search_url: "", filters: []) -%>
```

Comment:
Is this meant to be here?

Reply:
https://blog.kiprosh.com/allow-template-to-set-strict-locals/

So you can see what locals are allowed in a partial and assign defaults.